source/languages/en/riak/dev/advanced/write-once.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Write Once Buckets
+project: riak
+version: 2.1.0+
+document: tutorial
+audience: intermediate
+keywords: [developers, buckets]
+---
+
+Riak 2.1.0 introduces the concept of "write once" buckets; buckets whose entries
+are intended to be written exactly once, and never updated or over-written.
+Buckets of this type have the special feature that they circumvent the normal
+"coordinated put" path, which would otherwise result in a read on the
+coordinating vnode before the write. Avoiding coordinated puts results in higher
```

Comment:
I would edit this sentence to read: 

"Buckets of this type circumvent the normal "coordinated put" path..." 
Is it put or PUT?